docs/feature-flags.md

```diff
@@ -20,10 +20,6 @@ Subscription feature flags used by ARO-RP include:
   graduating features to production such that they only apply on our Red Hat
   engineering subscription.
 
-* Microsoft.RedHatOpenShift/SaveAROTestConfig: used on our test subscriptions to
```

Comment:
Make sure to also cleanup the following:
- https://msazure.visualstudio.com/AzureRedHatOpenShift/_git/WIKI-Designs?path=/Build%252Dout/ARM-Deployment-and-Management.md&_a=preview&version=GBmaster
- https://msazure.visualstudio.com/AzureRedHatOpenShift/_git/ARO-Pipelines?path=%2Farm%2Ffeature%2Fconfig.yaml&_a=contents&version=GBmaster

If I were you, I'd make a separate PR to remove `Microsoft.RedHatOpenShift/SaveAROTestConfig`

Reply:
Cleaned up the `config.yaml` on `ARO-Pipelines`. PR: https://msazure.visualstudio.com/AzureRedHatOpenShift/_git/ARO-Pipelines/pullrequest/9912014

Also cleaned up the wiki.